src/js/index.js

```diff
@@ -3,16 +3,4 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-/* eslint-disable */
-(function() {
-  /**
-   * Kick off all components
-   */
-  Alert.init();
-  Drawer.init();
-  Dropdown.init();
-  Modal.init();
-  Tabs.init();
-  FileInput.init();
-})();
-/* eslint-enable */
+export * from './components/alert.js';
```

Comment:
Along the same lines as my comment on `alert.js` and import/export strategy. Not right now, but eventually we will need to import all of our new js components here and then do some sort of set-up/initialization stuff in this file. For example implementing some sort of opt-in/opt-out auto-initialization functionality.

With that in mind we definitely need named references to the components so that we can do stuff (like auto-initialize them) with them.

```js
import Alert from './components/alert';
import OtherComponent from './components/other-components';

// Potential other set up code that needs to call Alert and OtherComponent eg. auto-initialize.

// Finally export an object under the Rivet namespace specified in our Rollup.js config
export default { Alert, OtherComponent };
```